packages/web-app-ocm/src/extensions.ts

```diff
@@ -69,6 +72,18 @@ export const extensions = () => {
         componentType: 'button',
         class: 'oc-files-actions-open-file-remote'
       }
-    }
+    },
+    ...((userStore.user && [
+      {
+        id: appInfo.id,
+        type: 'appMenuItem',
+        label: () => appInfo.name,
+        color: appInfo.color,
+        icon: appInfo.icon,
+        priority: 10,
+        path: urlJoin(appInfo.id)
```

Comment:
Okay for now for me, but I somewhat dislike having to define the same name, color, icon multiple times - I assume most of the time falling back to the global definition for the app would work as well

What do you think?

I think I've built this kind of fallback mechanism for file icons that are falling back to the app icon...

Reply:
As discussed: let's leave it as it is for now. It's not that trivial to build a fallback mechanism here because we don't know which app registers which menu item.